src/pages/Wallet/WalletOverview/components/AvailableAssetsTable/AvailableAssetsTable.tsx

```diff
@@ -36,6 +37,9 @@ type AvailableAssetsTableProps = {
   pooledTickers?: Set<string>;
 };
 
+// These are the currencies we want to show at zero balance
+const defaultCurrencies: CurrencyExt[] = [InterBtc, Interlay, KBtc, Kintsugi, Kusama, Polkadot];
```

Comment:
There is `NATIVE_CURRENCIES` that you can use here

Reply:
Nice, thanks—we may want to extend this (e.g. include USDT) but this works great for now